aviatrix/resource_aviatrix_transit_gateway_peering.go

```diff
@@ -136,6 +136,24 @@ func resourceAviatrixTransitGatewayPeering() *schema.Resource {
 					"Only valid when the two transit gateways are each launched in Insane Mode and in the same cloud type. " +
 					"Available as of provider version R2.22.2+.",
 			},
+			"enable_jumbo_frame": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+				Description: "Enable jumbo frame for over private peering with Edge Transit",
+			},
+			"src_wan_interfaces": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Description:  "Select src Edge Transit WAN interface(s) to attach, comma separated",
+				ValidateFunc: validateWanInterfaces,
+			},
+			"dst_wan_interfaces": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Description:  "Select dst Edge Transit WAN interface(s) to attach, comma separated",
+				ValidateFunc: validateWanInterfaces,
+			},
```

Comment:
Same comment about src and dst interfaces, what does this mean? do the descriptions give enough information for the consumers to understand what this is?

Reply:
Src interface is interface endpoint at the local gateway where the peering originates, and dst interface is the interface end point at the remote peer where this peering terminates.